Example/AccessibilitySnapshot/SwiftUIView+EmbedInHostingController.swift

```diff
@@ -0,0 +1,30 @@
+//
```

Comment:
I know y'all just got rid of this, but I brought it back because the navigation buttons were not being rendered when using only the SwiftUI views.

Reply:
Hmm, they weren't being rendered at all? The only thing that should be different between these is how the view is sized (setting the `frame` vs. `bounds`). Might affect the safe area insets, but everything should still be rendered.

Reply:
I'm sorry, I misspoke. The button is getting rendered but it is not included as an accessibility element

![testNavigationStack 393x852-16-4-3x](https://github.com/cashapp/AccessibilitySnapshot/assets/1452879/4e2cd3cd-6deb-4a12-81bf-b60de8702406)

Changing this code from `bounds` to `frame` resolves the issue: https://github.com/cashapp/AccessibilitySnapshot/blob/master/Sources/AccessibilitySnapshot/SnapshotTesting/SnapshotTesting%2BSwiftUI.swift#L68